client/src/holdings/AddTradeDialog.tsx

```diff
@@ -78,7 +65,7 @@ export function AddTradeDialog({
       category: 'BUY',
       date: new Date().toISOString(),
       broker: holding.trades.slice(-1)[0]?.broker ?? '',
-      priceCurrency: holding.currency,
```

Comment:
What happens if there is no previous trade, surely the priceCurrency can't be an empty string? Cos then the dropdown wouldn't work presumably. If this is the case then the default value when there's no trades should be holding.currency instead of ''

Reply:
Ok - makes sense